css/properties/text-align-last.json

```diff
@@ -80,7 +80,11 @@
               }
             ],
             "ie": {
-              "version_added": false
+              "version_added": true,
+              "notes": [
+                "IE only supports text-align-last when text-align is set to justify.",
```

Comment:
I think it would be nice to add some formatting, like this: `"IE only supports <code>text-align-last</code> when <code>text-align</code> is set to <code>justify</code>."`